pkg/sql/compile/cnserviceDriver.go

```diff
@@ -258,6 +264,9 @@ func sendBackBatchToCnClient(ctx context.Context, b *batch.Batch, messageId uint
 		m.Id = messageId
 		m.Data = encodeData[start:end]
 		m.Sid = uint64(sid)
+		m.Checksum = crc32.ChecksumIEEE(m.Data)
+		m.Sequence = mHelper.sequence
+		mHelper.sequence++
```

Comment:
Here is a concurrency scenario where you need to consider concurrency security, as well as performance